src/vma/util/sysctl_reader.h

```diff
@@ -140,6 +146,20 @@ public :
 		return &tcp_mem;
 	}
 
+	const tcp_keepalive_info *get_tcp_keepalive_info(bool update = false)
+	{
+		static tcp_keepalive_info info = {7200, 75, 9};
```

Comment:
use macros for those values, we have:

#define TCP_KEEPIDLE_DEFAULT 7200000UL /* Default KEEPALIVE timer in milliseconds */

#define TCP_KEEPINTVL_DEFAULT 75000UL /* Default Time between KEEPALIVE probes in milliseconds */

#define TCP_KEEPCNT_DEFAULT 9U /* Default Counter for KEEPALIVE probes */

Reply:
I am not sure we need special macro for one time initialization in function that should initialize.

Reply:
I agree. One more argument against this suggestion is to avoid creating an unnecessary coupling.